bin/aws-cloudformation-create-stack-policy

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env node
+let _ = require('lodash-firecloud').default;
+let fs = require('fs');
+
+let createStackPolicy = async function({cfnTemplatePath}) {
+  let cfnTemplate = JSON.parse(fs.readFileSync(cfnTemplatePath));
+  let resources = _.keys(cfnTemplate.Resources);
+  let retainedResources = _.filter(resources, function(resourceName) {
+    return _.get(cfnTemplate.Resources[resourceName], 'DeletionPolicy') === 'Retain';
+  });
```

Comment:
just nit-picking: doesn't this work if you ignore the "wasted" memory?
`let retainedResources = _.keys(_.pickBy(cfnTemplate.Resources, {DeletionPolicy: 'Retain'}));`